plugin.audio.100fm/addon.py

```diff
@@ -0,0 +1,44 @@
+import sys
+import routing
+from api import API
+from xbmc import Player, sleep, translatePath, getCondVisibility
+from xbmcgui import getCurrentWindowDialogId
+from xbmcaddon import Addon
+from player import PlayerWindow
+
+ADDON = Addon()
+ADDON_HANDLE = int(sys.argv[1])
+ADDON_ID = ADDON.getAddonInfo('id')
+ADDON_NAME = ADDON.getAddonInfo('name')
+ADDON_DATA_FOLDER = translatePath(ADDON.getAddonInfo('profile')).decode('utf-8')
+PLUGIN = routing.Plugin()
+
+
+@PLUGIN.route('/')
+def index():
+    """
+    Main add-on popup
+    :return:
+    """
+    player_window = PlayerWindow()
+    api = API()
+
+    # stop player if track is not part of this addon:
+    if Player().isPlaying() and '.100fm.' not in Player().getPlayingFile():
```

Comment:
There are some animations & info which need to be displayed only if 100fm tracks are being played